src/Arcus.Observability.Tests.Integration/Serilog/TelemetryTypeFilterTests.cs

```diff
@@ -338,167 +338,6 @@ public void LogCustomMetric_WithTelemetryTypeFilterOnDifferentTyp_DoesNotFilterO
             }
         }
 
-        [Fact]
-        public void LogRequest_WithTelemetryTypeFilter_FilersInRequest()
```

Comment:
why are these tests no longer necessary ?

Reply:
Because they tested now-deleted deprecated log extensions. Those extensions are not there anymore, so we can't test them. 😄